roles/openshift_gcp/files/openshift-bootstrap-controller-policy.yaml

```diff
@@ -0,0 +1,10 @@
+kind: ClusterRoleBinding
```

Comment:
Move to `openshift_bootstrap_autoapprover` role.  This is useful outside of gcp!